include/tbb/parallel_execution_tbb.h

```diff
@@ -30,73 +30,124 @@
 
 namespace grppi{
 
-/** @brief Set the execution mode to parallel with threading building blocks
- *    (TBB) framework implementation
+/** 
+ \brief TBB parallel execution policy.
```

Comment:
I use TBB in the short description and "Intel TBB" in the long description.

I think that should be enough.